### PR TITLE
chore(docs): add rules description

### PR DESCRIPTION
**Pull request by aslafy-z**
##### Description

These where the only rule description missing.

##### Type of change

- [ ] Bug fix :bug:
- [ ] New feature :sparkles:
- [ ] This change requires a documentation update :book:
- [ ] Breaking changes :warning:

##### Checklist:

- [x] I have :tophat: my changes (A 🎩 specifically includes pulling down changes, setting them up, and manually testing the changed features and potential side effects to make sure nothing is broken)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] The test coverage did not decrease
- [x] I have signed the appropriate [Contributor License Agreement](https://cla.shopify.com/)

### CODE REVIEW

File: auditors/image/image.go [hunk 1]
```diff
@@ -12,8 +12,11 @@ import (
 const Name = "image"
 
 const (
+	// ImageTagMissing occurs when the container image tag is missing
 	ImageTagMissing   = "ImageTagMissing"
+	// ImageTagIncorrect occurs when the container image tag does not match the one registered in podspec
```
Comment (genevieveluyt) [inline auditors/image/image.go:17]:
```suggestion
	// ImageTagIncorrect occurs when the container image tag does not match the user-provided value
```

Context: https://github.com/Shopify/kubeaudit/blob/master/docs/auditors/image.md#examples

**Review by genevieveluyt — COMMENTED** [review]
Hi @aslafy-z, Thank you for your contribution! Just one small change please

**Comment by aslafy-z** [comment]
Just applied it!

**Review by genevieveluyt — APPROVED** [review]
Thank you!

**Pull request merged**